src/main/resources/logback.xml

```diff
@@ -41,7 +47,18 @@
     <!--    <logger name="bio.terra.service.iam.sam.SamIam" level="debug" />-->
     <!--    <logger name="bio.terra.service.tabulardata.google.BigQueryProject" level="debug" />-->
 
+    <if condition="${SENTRY_ENABLED} == true">
```

Comment:
This looks like it will rely on an exact case sensitive match for this environment variable -- `True` or `TRUE` would not allow the condition to pass.  That's probably fine but I wanted to flag it!

Reply:
That's a good point.  I struggled a bit with finding a way for it to trigger off of a boolean value - I'm not sure this is the best way to do it. I think it should be okay? But, yeah, definitely good to note.